app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -19,61 +19,103 @@ inset_text:
     text: |-
       <p>There's still time to apply and start teacher training this September.</p>
       <p><a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find your postgraduate teacher training course now</a>.</p>
+keywords:
+  - becoming a teacher
+  - teacher training
+  - school direct
+  - pgce
+  - qts
 ---
 
 $still-time-to-apply$
 
-If you have a degree, or you’re currently studying for one, there are different routes you can take to getting [qualified teacher status](/what-is-qts) (QTS). You need QTS to work in the majority of schools in England including state maintained primary, secondary and special schools.
+You need a degree to teach in primary, secondary and special schools in England.
```

Comment:
Checking with Phoebe for wording about having any degree

Reply:
Phoebe's said she's not sure about the "any degree", because providers prefer people to have a degree in a relevant subject. Instead, she's advised something along the lines of:

"You need a bachelor's degree (honours) to teach in primary, secondary and special schools in England. This doesn't have to be a bachelor's degree in teaching. You'll need to demonstrate that you have adequate subject knowledge in your chosen subject."